bulma_form_builder.gemspec

```diff
@@ -26,5 +26,6 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "rake"
-  spec.add_development_dependency "rspec"
+  spec.add_development_dependency "rspec-rails", "~> 5.0"
+  spec.add_development_dependency "rspec-html-matchers", "~> 0.9"
```

Comment:
I don't want more gems to depend on.

Reply:
It is your call obviously but I think it is helpful to have some unit tests for the functionality. As development dependencies they do not become dependencies in production.